### ISSUE TITLE
sendChatAction & SendChatActionReq Fail

### ISSUE THREAD

**Issue by benkio** [apimorphism/telegramium#90]
At the moment, using the [`sendChatAction`](https://github.com/apimorphism/telegramium/blob/master/telegramium-core/src/main/scala/telegramium/bots/client/Methods.scala#L510) it will fail since the `SendChatActionReq` produce an empty JSON.
Instead. it should provide the options as in [telegram's api](https://core.telegram.org/bots/api#sendchataction).

This is the error message you get trying to run the method.

```
16:06:12.362 [scala-execution-context-global-93] ERROR telegramium.bots.high.BotApi - Telegram Bot API request failed: code=400 description="Bad Request: wrong parameter action in request" method=sendChatAction, JSON: 
{
  
}
```

I might try to fix it when I have time :wink:

**Comment by johnspade** [comment]
Thank you, I'll look into it

**Comment by johnspade** [comment]
Fixed in 4.51.1
